pkgs/by-name/ma/matrix-federation-tester/package.nix

```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, lib, fetchFromGitHub, nixosTests }:
+
+buildGoModule rec {
+  pname = "matrix-federation-tester";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "matrix-org";
+    repo = "matrix-federation-tester";
+    tag = "v${version}";
+    hash = "sha256-jf33eyjt4C5E1ite1NKkiQAwvB7zB9NQHdr+w/6nloQ=";
+  };
+
+  vendorHash = "sha256-UQUtD1zJaJf8UyRGUFSUHAB3OPfNFuVdc/oPUQrC6Do=";
+  strictDeps = true;
+
+  meta = {
+    mainProgram = "matrix-federation-tester";
+    homepage = "https://github.com/matrix-org/matrix-federation-tester";
+    description = "Tester for matrix federation written in golang";
+    maintainers = with lib.maintainers; [ ma27 ];
+  };
```

Comment:
Cannot be merged as-is, this is missing a license: https://github.com/matrix-org/matrix-federation-tester/issues/150